NDTensors/ext/NDTensorsMetalExt/linearalgebra.jl

```diff
@@ -1,15 +1,15 @@
-function NDTensors.qr(leaf_parenttype::Type{<:MtlArray}, A::AbstractMatrix)
-  Q, R = NDTensors.qr(NDTensors.cpu(A))
+function qr(A::Exposed{<:MtlArray})
+  Q, R = qr(expose(NDTensors.cpu(unexpose(A))))
```

Comment:
We could define `NDTensors.cpu(e::Exposed) = cpu(unexpose(e))` so we don't have to do this unwrapping here.